content/docs/start/experiments.md

```diff
@@ -120,6 +120,33 @@ Experiment results have been applied to your workspace.
 
 <details>
 
+### ℹ️ More information about (Hyper)parameters
+
+It's pretty common for data science projects to include configuration files that
+define adjustable parameters to train a model, adjust model architecture, do
+pre-processing, etc. DVC provides a mechanism for stages to depend on the
+specific variables from a file.
```

Comment:
> DVC provides a mechanism for stages to...

What are stages? First time that word is mentioned in this page. Let's generalize 🙂

Reply:
stages -> experiments in c0c233065